ecosystem/python/sdk/aptos_sdk/transactions.py

```diff
@@ -244,7 +244,7 @@ class ScriptArgument:
     U128: int = 2
     ADDRESS: int = 3
     U8_VECTOR: int = 4
-    BOOL: int = 5,
+    BOOL: int = (5,)
```

Comment:
I don't think this is supposed to have a comma

Reply:
I also don't think there should be a comma here, I'll fix it  in  next pr.